### PR TITLE
Updated code style to be PSR2 compliant

### PR DESCRIPTION
**Pull request by arjank**
Changes are whitespace changes (including newline changes) and string splits to reduce line length.

This fixes issue #116 (and overlaps with PR #117)

### CODE REVIEW

**Comment by tbarn** [comment]
Hey @arjank! Sorry about the mix up with who was working on this issue. We already reviewed #117 and was ready to merge yesterday as partial update to PSR-2 compliance. I know there are some conflicts now on `src/Client/Resources/keen-io-3_0.php`. I'm looking into them now.

**Comment by tbarn** [comment]
It seems there be as discrepancy around what line lengths should be. Tagging @Sekonda since he did the initial work on this.

**Comment by Sekonda** [comment]
@tbarn Is it possible to see what the conflicts are? 

[Standards say](http://www.php-fig.org/psr/psr-2/#overview) that lines must not be longer than 120 characters, which is what I had cut the lines at (depending on the word, it might be cut sooner). 

My editor (PHPStorm) also looks to have formatted the arrays different from before (key => values as now lined up with the rest of the array to make it easier to read). 

With the array being formatted a there could be different line lengths based of that.

File: src/Client/KeenIOClient.php [hunk 1]
```diff
@@ -53,8 +54,8 @@ public static function factory($config = array())
     {
         $default = array(
             'masterKey' => null,
-            'writeKey'  => null,
-            'readKey'   => null,
+            'writeKey' => null,
```
Comment (Sekonda) [inline src/Client/KeenIOClient.php]: In here is what I mean by the array being formatted. This line for example removes the formatting that was applied.

**Comment by tbarn** [comment]
@Sekonda can you see this? 

screen shot 2017-10-05 at 1 43 34 pm

If you click, "Resolve conflicts," you can see the conflicts in the GitHub web UI.

**Comment by Sekonda** [comment]
@tbarn I cannot, only people with write access can. Seems strange that the public can’t see the conflicts. 

385d9c99-01a1-420a-9453-4beedb292830

**Comment by tbarn** [comment]
Ugh, yeah, that's annoying.  

I created a gist with most of the conflicts: https://gist.github.com/tbarn/539ae8a37cd4d21a324803fe889d44e6

**Comment by Sekonda** [comment]
@tbarn most of them just look like formatting stuff. 

@arjank can you update your PR so that only the tests are compatible since #117 has already resolved the main package?

**Comment by arjank** [comment]
@tbarn @Sekonda I'll update the PR later today.

**Comment by arjank** [comment]
Merge conflicts have been resolved. There are no changes left in `src`.

**Comment by tbarn** [comment]
Thanks @arjank and @Sekonda! 

@arjank here's the form to fill out to get your shirt from your Hacktoberfest contribution: https://goo.gl/forms/w3HK9LWeau1UbZwl2

**Pull request merged**